### ISSUE TITLE
Installations instructions are marked up as a <figure>

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4583]
The installation instructions on the front page are marked up with a figure-element. They are not images but maybe even the most important text on the page for many. This can be improved by putting this content inside an element with different semantics (or no semantics) that don't confuse a user of Assistive Technology.

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos

**Comment by vhashimotoo** [comment]
Fixed by #4608
